### PR TITLE
Frontend v3

### PR DESCRIPTION
**Pull request by witkiewiczmikolaj**
Changed way of choosing action and target kind and some visual changes.

### CODE REVIEW

File: core/app/app.py [hunk 1]
```diff
@@ -22,8 +22,11 @@ def static_main():
     return render_template('index.html', module = module, action = action, target = target, button_clicked = button_clicked, languages = languages)
 
 @helbreder.route('/code')
```
Comment (wiktorkisielewski) [inline core/app/app.py:24]: please define what requests are allowed here:
`@helbreder.route('/code', methods=['???'])`

File: core/app/app.py [hunk 2]
```diff
@@ -22,8 +22,11 @@ def static_main():
...
-def code_outcome(): 
-    code = lang_gen(request.args.get('code'))
+def code_outcome():
+    try:
+        code = lang_gen(request.args.get('code'))
+    except:
```
Comment (wiktorkisielewski) [inline core/app/app.py:28]: what `exception` is expected here?
it **must be defined**

**Review by wiktorkisielewski — CHANGES_REQUESTED** [review]

File: core/app/app.py (hunk 1 above)
Comment (witkiewiczmikolaj) [reply core/app/app.py]: Resolved in commit: https://github.com/wiktorkisielewski/helbreder/pull/6/commits/ba6649d844725a3c9eb455619b7bb156fc70e920

File: core/app/app.py (hunk 2 above)
Comment (witkiewiczmikolaj) [reply core/app/app.py]: Resolved in commit: https://github.com/wiktorkisielewski/helbreder/pull/6/commits/ba6649d844725a3c9eb455619b7bb156fc70e920

File: core/app/templates/sql.py [hunk 3]
```diff
@@ -11,3 +12,4 @@ def get_lang_sql():
 def create_table():
     cur.execute('''CREATE TABLE IF NOT EXISTS LANGUAGES(id int, name varchar(255))''')
     c.commit()
+    c.close()
```
Comment (wiktorkisielewski) [inline core/app/templates/sql.py:15]: nice

**Review by wiktorkisielewski — APPROVED** [review]

**Pull request merged**